extension/src/popup/components/account/AccountAssets/index.tsx

```diff
@@ -83,13 +85,17 @@ export const AssetIcon = ({
   );
 };
 
+interface AccountAssetsProps {
+  assetIcons: AssetIcons;
+  sortedBalances: Array<any>;
+  setSelectedAsset?: (selectedAsset: string) => void;
```

Comment:
thoughts on this name change? from the AccountAssets POV the setSelectedAsset makes it seem like it's a `useState` var w/in AccountAssets
```suggestion
  handleAssetSelect?: (selectedAsset: string) => void;
```

Reply:
So this actually is a `useState` var, just being passed from the parent. Does that make it less confusing? Totally fine if it's still confusing, but I have used this naming convention in a few other places (for ex: https://github.com/stellar/freighter/blob/master/extension/src/popup/views/Account/index.tsx#L138)

I'm happy to create a ticket to change this naming convention if you feel strongly about it!

Reply:
ahh ok,  now that I have that way of thinking in my head of it's a useState from a higher component makes sense to leave 👍 thanks for explaining